packages/formatter-summary/package.json

```diff
@@ -0,0 +1,74 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "chalk": "^2.3.1",
+    "lodash": "^4.17.5",
```

Comment:
Can we change this to using only the method(s) that we are using instead of the whole lodash?